Rollbar/Abstraction_Common/JSONSupport.h

```diff
@@ -11,7 +11,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// JSON de/serialization protocol
-DEPRECATED_MSG_ATTRIBUTE("In v2, use RollbarJSONSupport instead.")
+//DEPRECATED_MSG_ATTRIBUTE("In v2, use RollbarJSONSupport instead.")
```

Comment:
Question: Why are we commenting the deprecated message here and many other places? Will these comments be removed in future?

Reply:
@mrunalk, the plan is to uncomment out these back when the official v2 is released and have sort of final v1 release we these deprecation warnings that would help to guide the SDK users during v1 to v2 transition.